docs/UserGuide.md

```diff
@@ -410,7 +421,8 @@ If the example was not executed successfully, the proper syntax of the `add` com
 `PHONE`
 * Phone number of the contact.
 * Only digits, '+', '-', ',' and spaces are allowed.
-* Any number of these characters are allowed.
+* The phone number must contain at least 3 characters.
+* Example: `+65-8765-4321, +97 6223 5555`.
```

Comment:
Currently, phone numbers are not validated extensively, e.g. `+++` is a valid input. Should we mention that as well?

Reply:
No, as it is not a bug to allow seemingly unsuitable values.

![image](https://github.com/AY2324S2-CS2103T-W12-3/tp/assets/65088428/517dec9e-050f-4e25-b71b-9cd05f01fb92)